src/multupd.jl

```diff
@@ -7,40 +7,44 @@
 #
 
 mutable struct MultUpdate{T}
-    obj::Symbol         # objective :mse or :div
-    maxiter::Int        # maximum number of iterations
-    verbose::Bool       # whether to show procedural information
-    tol::T              # change tolerance upon convergence
-    lambda::T           # regularization coefficient
+    obj::Symbol     # objective :mse or :div
+    maxiter::Int    # maximum number of iterations
+    verbose::Bool   # whether to show procedural information
+    tol::T          # change tolerance upon convergence
+    lambda_w::T     # L1 regularization coefficient for W
+    lambda_h::T     # L1 regularization coefficient for H
 
     function MultUpdate{T}(;obj::Symbol=:mse,
                             maxiter::Integer=100,
                             verbose::Bool=false,
                             tol::Real=cbrt(eps(T)),
-                            lambda::Real=sqrt(eps(T))) where T
+                            lambda_w::Real=zero(T),
```

Comment:
```suggestion
                            lambda_w=lambda,
```
would make things easier later on